docker-compose.yml

```diff
@@ -64,3 +64,16 @@ services:
       - "./docs/build/html:/usr/share/nginx/html"
     ports:
       - "8081:80"
+
+  dev-docs:
+    build:
+      context:  .
+      dockerfile: DockerfileDevDocs
```

Comment:
ik zou alleen de filename anders noemen. `dev-docs.dockerfile` ? `Dockerfile.dev-docs` ?

Reply:
Ik ben open voor de suggesties :) `Dockerfile.dev-docs` lijkt me goed. In Airflow we hebben `Dockerfile-*` files, dus misschien `Dockerfile-dev-docs` is beter.

Reply:
Allemaal prima!

Reply:
`Dockerfile-dev-docs` it is.

Reply:
dev-docs/Dockerfile werkt ook, want de build context wordt in docker-compose.yml ingesteld. Nadeel is dat je dan niet docker build . kunt doen in dev-docs.

Reply:
En dat betekend dat Jenkins kan image niet bouwen. Dus blijft het `Dockerfile-dev-docs` in de root.